code/modules/unit_tests/serving_tray.dm

```diff
@@ -5,9 +5,9 @@
 	var/mob/living/carbon/human/human = allocate(/mob/living/carbon/human)
 	var/obj/structure/table/the_table = allocate(/obj/structure/table)
 	var/obj/item/storage/bag/tray/test_tray = allocate(/obj/item/storage/bag/tray)
-	var/obj/item/reagent_containers/food/banana = allocate(/obj/item/reagent_containers/food/snacks/rationpack)
+	var/obj/item/reagent_containers/food/banana = allocate(/obj/item/food/rationpack)
 	var/obj/item/food/the_bread = allocate(/obj/item/food/breadslice)
-	var/obj/item/reagent_containers/food/sugarcookie = allocate(/obj/item/reagent_containers/food/snacks/sugarcookie)
+	var/obj/item/reagent_containers/food/sugarcookie = allocate(/obj/item/food/cookie/sugar)
```

Comment:
I'm not sure why we have two old food in this test, but *one* of them should stay as an `/obj/item/reagent_containers/food` until everything is ported over. Perhaps leave a comment about that as well.

Reply:
oh wait